bindings/python/tests/test_transaction.py

```diff
@@ -0,0 +1,73 @@
+from pathlib import Path
+
+try:
+    import pytest
+except ImportError:
+    HAVE_PYTEST = False
+else:
+    HAVE_PYTEST = True
+
+import dlite
+
+
+# Configure paths
+thisdir = Path(__file__).parent.absolute()
+
+#dlite.storage_path.append(thisdir / 'entities' / '*.json')
+dlite.storage_path.append(thisdir / '*.json')
+Person = dlite.get_instance('http://onto-ns.com/meta/0.1/Person')
+
+
+person = Person(dims={"N": 4})
+person.name = "Knud H. Thomsen"
+person.age = 30
+person.skills = ["writing", "humor", "history", "human knowledge"]
+
+# Update person 6 times and create a snapshot before each change
+for i in range(6):
+    person.snapshot()
+    person.age += 5
+
+assert person.age == 60
+assert person.get_snapshot(0).age == 60
+assert person.get_snapshot(1).age == 55
+assert person.get_snapshot(6).age == 30
```

Comment:
```suggestion
for i in range(7):
    assert person.get_snapshot(i).age == 30 + (6 - i) * 5
```

Reply:
Is such a for loop more readable than testing for explicit values?

Reply:
Perhaps not - I suppose it's a matter of taste. Personally I like it most like this simply due to the fact that one can see the intended way the value is calculated - meaning the test as a whole becomes more transparent.